src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -231,7 +231,7 @@ Task IDurableOrchestrationClient.RaiseEventAsync(string instanceId, string event
                 throw new ArgumentNullException(nameof(eventName));
             }
 
-            return this.RaiseEventInternalAsync(this.client, this.TaskHubName, instanceId, eventName, eventData);
+            return this.RaiseEventInternalAsync(this.client, this.TaskHubName, instanceId, eventName, eventData, this.durabilityProvider.CheckStatusBeforeRaiseEvent);
```

Comment:
Rather than passing this flag as a method argument, you can reduce the size of this change by having `RaiseEventInternalAsync` query this new property directly.

Reply:
I think I will leave it as it is since it allows me to always disable the status check when the function is called from CleanEntityStorageAsync (in that context the check is always superfluous since the entity status was already checked earlier).